modules/aleph/src/Aleph/AlephRequest.php

```diff
@@ -0,0 +1,396 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+/**
+ * Class AlephRequest.
+ *
+ * Entity with setters/getters for Z37 (REQUESTS) in Aleph.
+ *
+ * The Z37 contains requests information.
+ * The Z37 record functions both for requests for
+ * loaned items (waiting requests), and for requests for delivery of available
+ * items. It is not necessarily expected that a library will convert requests
+ * when installing the ALEPH system. In many cases, the library will inform
+ * patrons that there is a system switchover, that outstanding requests are
+ * not retained, and must be re-entered after switchover.
+ *
+ * https://www.obvsg.at/uploads/media/Z37.pdf
+ *
+ * @package Drupal\aleph\Aleph
+ */
+class AlephRequest {
+
+  /**
+   * Z37-STATUS.
+   * Hold request status.
+   *
+   * A = Active (newly opened, not yet processed).
+   *
+   * S = Hold Shelf - Request has been trapped, patron has been notified that
+   * the requested material is ready to be picked up.
+   *
+   * W = Waiting. System generated when the "print call slip" service
+   * (b- cir-12) was run, and the system detected that there was not an
+   * available copy
+   *
+   * @var string
+   */
+  protected $status;
+
+  /**
+   * Z37-ITEM-SEQUENCE.
+   * Item sequence number.
+   *
+   * The number is within one ADM number.
+   *
+   * @var string
+   */
+  protected $itemSequence;
+
+  /**
+   * Z37-SEQUENCE.
+   * Sequence number of a request.
+   *
+   * Used to distinguish between multiple requests placed on the same item.
+   *
+   * @var string
+   */
+  protected $sequence;
+
+  /**
+   * Z37-ID.
+   * Patron’s ID.
+   *
+   * @var string
+   */
+  protected $id;
+
+  /**
+   * Z37-PICKUP-LOCATION.
+   * User’s pickup sub-library.
+   *
+   * The default is the user's home sub-library.
```

Comment:
That's describing the backend again. All we need to know is that it's the pickup location.